### PR TITLE
add kubectl_test to arm64 e2e test suit

### PR DESCRIPTION
**Pull request by zhlhahaha**
**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by zhlhahaha** [comment]
verify if pull-kubevirt-e2e-arm64 CI test works well
/test pull-kubevirt-e2e-arm64

**Comment by zhlhahaha** [comment]
CC: @rmohr

File: tests/framework/checks/skips.go [hunk 1]
```diff
@@ -86,3 +86,18 @@ func SkipTestIfNotSEVCapable() {
...
 }
+
+func SkipTestIfNotEnoughSchedulableNode(nodeCount int) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	util.PanicOnError(err)
+	nodes := util.GetAllSchedulableNodes(virtClient)
+	schedulableNodesNum := len(nodes.Items)
+	if schedulableNodesNum < nodeCount {
+		msg := fmt.Sprintf(
+			"no enough Schedulable node: expected %v nodes, but got %v",
```
Comment (rmohr) [inline tests/framework/checks/skips.go:97]:
```suggestion
			"skipping due to not enough Schedulable nodes: tes requires %v nodes, but got %v",
```

File: tests/operator_test.go [hunk 2]
```diff
@@ -1216,6 +1217,9 @@ spec:
 				}),
 			table.Entry("[test_id:6308] daemonsets",
 				func() {
+					// When only have one node, updating virt-hander would get following error
+					// "Unable to mark node as unschedulable"
```
Comment (rmohr) [inline tests/operator_test.go:1221]: Could you explain that a little bit? I would expect that we can update virt-handler without issues and I can't see where we are setting the node to unschedulable.

File: tests/operator_test.go [hunk 3]
```diff
@@ -2229,6 +2239,7 @@ spec:
 	Context("With PrometheusRule Enabled", func() {
 
 		BeforeEach(func() {
+			tests.SkipIfARM64("Currently not works on Arm64")
```
Comment (rmohr) [inline tests/operator_test.go:2242]: Could you rephrase these:

```suggestion
			tests.SkipIfARM64("Currently not supported on arm64")
```

**Review by rmohr — CHANGES_REQUESTED** [review]

File: tests/operator_test.go (hunk 2 above)
Comment (zhlhahaha) [reply tests/operator_test.go:1221]: Hi Roman, I did more investigation on it. If I only test `test_id:6308`, it would pass, but when I test both `test_id:6256` and `test_id:6308` on a one-node k8s cluster, the tests get fail. Test `test_id:6308` restarts virt-hander, but virt-hander fail to start with following error message:
```
{"Unable to mark node as unschedulable":"nodes \"$(NODE_NAME)\" not found","component":"virt-handler","level":"error","pos":"virt-handler.go:175","timestamp":"2022-04-15T02:05:23.421281Z"}
```
And I use `ps aux` to see the command of virt-hander, it shows 
```
virt-handler --port 8443 --hostname-override $(NODE_NAME) --pod-ip-address $(MY_POD_IP) --max-metric-requests 3 --console-server-port 8186 --graceful-shutdown-seconds 315 -v 2
```
NODE_NAME and MY_POD_IP does not have correct value. It seems that NODE_NAME is failed to get value from spec.nodeName. 
```
        containerEnv := []corev1.EnvVar{
                {
                        Name: "NODE_NAME",
                        ValueFrom: &corev1.EnvVarSource{
                                FieldRef: &corev1.ObjectFieldSelector{
                                        FieldPath: "spec.nodeName",
                                },
                        },
                },
                {
                        Name: "MY_POD_IP",
                        ValueFrom: &corev1.EnvVarSource{
                                FieldRef: &corev1.ObjectFieldSelector{
                                        FieldPath: "status.podIP",
                                },
                        },
                },
        }
```
Any suggestion?
